command/server/server.go

```diff
@@ -67,42 +67,42 @@ func setFlags(cmd *cobra.Command) {
 	cmd.Flags().IntVar(
 		&params.leveldbCacheSize,
```

Comment:
Consider rename those flags to `--db.cache, --db.xxx` or `--kvdb.xxx` ?
These word feel more like the same level as the 'network', etc.

Reply:
It was a recommendation, not a request change.
So I'll set the conversation to resolved.